tests/models/test_baseoperator.py

```diff
@@ -902,3 +909,13 @@ def add(x, y):
 
     dependants = list(gen_result.operator.iter_mapped_dependants())
     assert dependants == [add_result.operator]
+
+
+class TestBaseDeferrableOperator:
+    def test_load_default_deferrable(self) -> None:
+        class EmptyDeferrableOperator(BaseDeferrableOperator):
+            def execute(self, _):
+                pass
+
+        empty_deferrable_operator = EmptyDeferrableOperator(task_id="deferrable_operator")
+        assert empty_deferrable_operator.deferrable is DEFAULT_DEFERRABLE
```

Comment:
```suggestion
def test_load_default_deferrable() -> None:
    class EmptyDeferrableOperator(BaseDeferrableOperator):
        def execute(self, _):
            pass

    empty_deferrable_operator = EmptyDeferrableOperator(task_id="deferrable_operator")
    assert empty_deferrable_operator.deferrable is DEFAULT_DEFERRABLE
```

Don’t need the class

Reply:
Thanks! Just applied it.